packages/docusaurus/src/server/i18n.ts

```diff
@@ -7,45 +7,76 @@
 import {I18n, DocusaurusConfig, I18nLocaleConfig} from '@docusaurus/types';
 import path from 'path';
 import {normalizeUrl} from '@docusaurus/utils';
+import {getLangDir} from 'rtl-detect';
+import {NODE_MAJOR_VERSION} from '../constants';
+import chalk from 'chalk';
 
-export function defaultLocaleConfig(locale: string): I18nLocaleConfig {
+function getDefaultLocaleLabel(locale: string) {
+  if (NODE_MAJOR_VERSION >= 14) {
+    // @ts-expect-error: why?
+    return new Intl.DisplayNames([locale], {type: 'language'}).of(locale);
+  }
+  return locale;
+}
```

Comment:
@longlho particularly not sure of this part.

It seems `Intl.DisplayNames` does not exist on Node 13. Can you help me figure out where to check the NodeJS support of Intl apis?

Also not sure how to make TS happy

Reply:
`Intl.DisplayNames` is ES2021 which is not yet published so it won't be in TS native type def for a while

Reply:
Please ignore the rest of the diff, this is mostly for this code that I pinged you 😆

Thanks, didn't know. 
So is it safe to consider this API is available in Node14? Not sure where to check that exactly

Reply:
yeah as long as MDN says it is then you're good. I wouldn't check for node version but rather check for `if (!!Intl.DisplayNames)` or something like that

Reply:
Thanks, will do that.

It seems I can get this kind of infos on the v8 site directly: https://v8.dev/features/intl-displaynames

Seems supported since Node 14+, so I think we'd rather recommend using Node 14 and not just node >= 13 for Docusaurus i18n, as it's also a LTS

Thanks for the review

Reply:
Just for curiosity, why is this stage 4 in 2020 and not in ES2020?

Apparently, it's not even ES2021 but 2022?
https://tc39.es/ecma402/#intl-displaynames-objects